lib/plugins/config/settings/config.metadata.php

```diff
@@ -230,6 +230,9 @@
 $meta['search_fragment'] = array('multichoice','_choices' => array('exact', 'starts_with', 'ends_with', 'contains'),);
 $meta['trustedproxy'] = array('regex');
 
+$meta['_feature_flags'] = ['fieldset'];
```

Comment:
Does it make sense to add a new config section for this? I would have added it to the "Advanced" section. OTOH, if we do add a new section, it might make sense to rearrange some other settings as well?

Reply:
I created that Feature Flag section to indicate that these Feature Flags are about to be dropped again soon. Maybe already before the next release, maybe only for the release after that.
To my understanding, Feature Flags are a specific pattern for rolling out features safely. https://en.wikipedia.org/wiki/Feature_toggle

It is not intended to permanently provide the ability to turn these features off, only until we are sure that we are not breaking too much.

If we want to make this a permanent toggle, then it should indeed be integrated into the "advanced" section. But I'm not sure I see the need for that?

Reply:
Makes sense 👍